OpenRA.Mods.Common/Projectiles/Bullet.cs

```diff
@@ -101,13 +101,36 @@ public class BulletInfo : IProjectileInfo
 		[Desc("Altitude above terrain below which to explode. Zero effectively deactivates airburst.")]
 		public readonly WDist AirburstAltitude = WDist.Zero;
 
+		[Desc("Length of the contrail (in ticks).")]
 		public readonly int ContrailLength = 0;
-		public readonly int ContrailZOffset = 2047;
-		public readonly Color ContrailColor = Color.White;
-		public readonly bool ContrailUsePlayerColor = false;
+
+		[Desc("Delay of the contrail (in ticks).")]
 		public readonly int ContrailDelay = 1;
+
+		[Desc("Offset for contrail's Z sorting.")]
+		public readonly int ContrailZOffset = 2047;
+
+		[Desc("Width of the contrail.")]
```

Comment:
```suggestion
		[Desc("Thickness of the emitted line.")]
```

Try to not just repeat variable name when describing it.